tfjs-backend-webgpu/src/binary_op_util.ts

```diff
@@ -59,6 +60,9 @@ const ADD = 'return a + b;';
 const COMPLEX_MULTIPLY_REAL = 'return areal * breal - aimag * bimag;';
 const COMPLEX_MULTIPLY_IMAG = 'return areal * bimag + aimag * breal;';
 const DIV = 'return a / b;';
+const ELU_DER = 'return select(a * (b + 1.0), a, b >= 0.);';
```

Comment:
I see webgl is using `b >= 1.0` https://github.com/tensorflow/tfjs/blob/master/tfjs-backend-webgl/src/kernels/EluGrad.ts#L24 . However, you are using `b >= 0.0`. Which one is correct?

Reply:
On webgl backend, the non PACKED and PACKED versions are different, https://github.com/tensorflow/tfjs/blob/b6a5e25f2517149a13383aaac641f119763968b0/tfjs-backend-webgl/src/kernels/EluGrad.ts#L25
I think the PACKED version is correct, but is not 100% sure.

Reply:
According to http://bytedeco.org/javacpp-presets/tensorflow/apidocs/org/bytedeco/tensorflow/EluGrad.html, b >= 0. is correct. 
Please also fix the WebGL unpacked version, and I think we also need some test cases to cover.

Reply:
Fixed them and added case in https://github.com/tensorflow/tfjs/pull/7251

Reply:
Thanks! Please hold this a bit until #7251 is reviewed.